go.mod

```diff
@@ -156,17 +155,11 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
 
 // TODO: remove when middleware will be implemented
-replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.2.1-hooks
+replace github.com/cosmos/ibc-go/v2 => github.com/crypto-org-chain/ibc-go/v2 v2.0.0-hooks-2
 
-replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.1
+replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.2
 
-replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.3-patched
+replace github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.6.1-0.20211206083034-fc6185e4eb5e
 
-// TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
-replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.2
-
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-4
-
-// FIXME: update after PR merged: https://github.com/PeggyJV/gravity-bridge/pull/182
-// https://github.com/crypto-org-chain/gravity-bridge/tree/cronos
-replace github.com/peggyjv/gravity-bridge/module => github.com/crypto-org-chain/gravity-bridge/module v0.1.22-0.20211011065300-a09cf050d304
\ No newline at end of file
+// FIXME: update after PR merged: https://github.com/PeggyJV/gravity-bridge/pull/310
+replace github.com/peggyjv/gravity-bridge/module => github.com/yihuang/gravity-bridge/module v0.1.22-0.20211206072459-b197433bdb33
```

Comment:
which repo did you use for this version?

Reply:
https://github.com/yihuang/gravity-bridge/tree/update-deps
it seems merged now.

Reply:
okay if you can sync with upstream, then I can close my PR (#239) 
Also remove the comment

Reply:
yeah, I changed to upstream main branch.